### ISSUE TITLE
CLI to update metadata of zarr stores

### ISSUE THREAD

**Issue by ieivanov** [czbiohub-sf/iohub#210]
@ziw-liu and I discussed the need to update metadata of a zarr store after it has been created. In `shrimPy` we have an [update_scale_metadata](https://github.com/czbiohub-sf/shrimPy/blob/main/mantis/cli/update_scale_metadata.py) CLI call which we use frequently - it allows us to update the scale metadata by calling `Position.set_transform` on all positions in a zarr store. @ziw-liu also brought up the need to update the channel names in a zarr store, and I know that @Soorya19Pradeep has also had similar needs - this can be done using `Position.rename_channel`.

We are asking other users if they would prefer to have CLI calls in `iohub` to update these metadata or if the existing python API is sufficient to do that, say in a python script. 

Separately, are there other metadata fields which you have had the need to update? Is there a need for more general methods to update the ome-zarr metadata (outside of channel names and data scale)? @ziw-liu brought up the point that the metadata tree is relatively deep and a general `update-metadata` method will be difficult to maintain.

@edyoshikun @talonchandler @JoOkuma

**Comment by ieivanov** [comment]
I have occasionally wanted to update the mix/max/start/end values of the contrast limits, such that data will open up in napari looking nice without having to adjust the contrast slider every time. Minor annoyance for now.

**Comment by JoOkuma** [comment]
There's a [napari-metadata](https://github.com/andy-sweet/napari-metadata) for editing ome-zarr metadata, but it doesn't allow editing all the fields, and I'm not sure if it works with HCS files.

I usually change `start` and `end` as Ivan. To do this, I use a text editor, but this doesn't scale to multiple plates.

**Comment by edyoshikun** [comment]
I regularly use the shrimPy `update-scale-metadata` especially with Falcon datasets since we don't have a way to tell it that it's a 60x vs 20x objective when we swap them, so inherently the pixel sizes in XY are wrong. 
Also, currently use it for virtual staining since the predictions default scale is (1,1,1,1,1). 

This probably is better suited as an iohub CLI than shrimPy.

**Comment by talonchandler** [comment]
I agree that modifying scale, channel names, and contrast limits are good candidates for CLI shortcuts in `iohub`.

**Comment by ziw-liu** [comment]
@ieivanov Are these all implemented either in iohub or biahub?

**Comment by ieivanov** [comment]
#228 introduced a `set-scale` CLI in iohub, which was certainly needed.
#232 also introduced a CLI to rename wells, along with iohub API to do at the `Position` level.

We do have API to rename channels. There is no CLI for that - I think that's OK for now?

We don't yet have API to update the mix/max/start/end values in the ome metadata - that would be useful. It would also be nice to have API to set the "active" channels and the channel color. That we currently do by directly modifying the json, which is not great. These are all relate to setting the omero channels setting. Maybe we can have some soft of API for that?

**Comment by ziw-liu** [comment]
Thanks, I'm moving this to v0.3 since they are not blocking.
